src/main/java/org/jitsi/videobridge/Endpoint.java

```diff
@@ -1599,6 +1617,15 @@ public Transceiver getTransceiver()
         return transceiver;
     }
 
+    /**
+     * @return this {@link Endpoint}'s bitrate controller
+     */
+    @Nonnull
+    public BitrateController getBitrateController()
```

Comment:
I don't see anything using this?  I'd rather not expose this and keep it encapsulated.

Reply:
@bbaldino the reason why I exposed it, it's because in my code which uses `JVB` as a library I only have an access to endpoint instance and few other public classes and I wanted having access to `Endpoint` object to understand when the set of forwarded endpoints is changed. 
So without access to bitrate controller I can't listen it's events.

So if you think `BitrateController` must stay private to endpoints, would you mind if I add an forwarded endpoint changed event on endpoint level? I mean adding `EventHandler` for endpoint and putting `forwardedEndpointChanged` here.
I think now that I would prefer `EventHandler` approach, because I've just recently realized I also want an event when endpoint is selected by some other endpoint. Currently, endpoint just directly send message to a message transport, but it would be really nice to have an event here.
And once I written all of this I realized that with `EventHandler` on endpoint level I could reduce diff significantly... so I'll will create another PR to show an idea.

Reply:
Yeah, I think that sounds good.